tests/next_tests/integration_tests/cases.py

```diff
@@ -192,13 +192,13 @@ class UniqueInitializer(DataInitializer):
     data containers.
     """
 
-    start: int = 0
+    start: int = 1
```

Comment:
I might have added a test that relies on start being zero, which is a mistake. Let me know if you see a problem.